docs/distributing-an-application/installing-with-swarm.md

```diff
@@ -25,6 +25,89 @@ Quick install:
 curl -sSL https://get.replicated.com/swarm-init | sudo bash
 ```
 
-## Installing Behind A Proxy
+### Installing Behind A Proxy
 
 Proxy support for Swarm will be included in a future release of Replicated.
+
+### Uninstall
+
+__Stacks__
+
+
+```shell
+$ docker stack ls
+```
+
+Using that output, run:
+
+```shell
+$ docker stack rm <stack_name>
+```
+
+for each stack with `replicated` in the name. Some portions of the stack deletion might fail, we will get back to those.
+
+__Services__
+
+```shell
+$ docker service ls
+```
+
+Using that output, run:
+
+```shell
+$ docker service rm <service_name>
+```
+
+for each service with `premkit`, `statsd`, or `replicated` in the name.
+
+__Stacks Again__
+
+```shell
+$ docker stack ls
+```
+
+Using that output, run:
+
+```shell
+$ docker stack rm <stack_name>
+```
+
+for each stack with `replicated` in the name.
+
+
+__Lingering Containers__
+
+
+```shell
+$ docker ps
```

Comment:
sb: docker ps -a

I had a stopped piper container that prevented me from removing a volume